src/porepy/geometry/point_in_polyhedron_test.py

```diff
@@ -33,29 +32,27 @@ def __init__(
         self.tol: float = tol
 
     def solid_angle(self, R: np.ndarray[Any, np.dtype[np.float64]]) -> float:
-        """Implements an analytical expression for the solid angle.
+        """ Implements an analytical expression for the solid angle.
 
         The solid angle is described by a plane triangle surface S at some arbitrary
         point P. It is the measure of the intersection of the three-dimensional unit
         sphere centered at P and the conic hull defined by the vectors in R.
 
         The implementation is based on equation (6) in reference: Robust inside-outside
-            segmentation using generalized winding numbers
-            (https://doi.org/10.1145/2461912.2461916)
+        segmentation using generalized winding numbers
+        (https://doi.org/10.1145/2461912.2461916)
 
         Args:
-            R (np.ndarray, num_pt x 3): Translated triangle's points at origin (0,0,0).
-            The original triangle's points need to be translated by subtracting the
-            arbitrary point P.
+            R (shape=(num_pt,3)): Translated triangle's points at origin (0,0,0).
+                The original triangle's points need to be translated by subtracting the
+                arbitrary point P.
 
         Raises:
-            ValueError: If the origin [0,0,0] point:
-                coincides with a vertex
-                is collinear with the vertices
-                is coplanar with the vertices
+            ValueError: If the origin [0,0,0] point coincides with a vertex,
+                is collinear with the vertices, or is coplanar with the vertices
```

Comment:
Let the line run to at most 88 characters before introducing the line break, but don't break the line before that unless you have a too long word. This applies as a general comment, I see some other places you want to fix this.

There is a nice plugin in VSCode which will do this for you (rewrap it is called), if you use pycharm, it may have something similar. It is definitely worth looking for this.